packages/docusaurus-mdx-loader/src/loader.ts

```diff
@@ -61,15 +62,14 @@ const {
 const DEFAULT_OPTIONS: MDXOptions = {
   admonitions: true,
   rehypePlugins: [],
-  // @ts-expect-error: TODO
   remarkPlugins: [emoji, headings, toc],
   beforeDefaultRemarkPlugins: [],
   beforeDefaultRehypePlugins: [],
 };
 
 const compilerCache = new Map<string | Options, [Processor, Options]>();
 
-export type MDXPlugin = PluggableList;
+export type MDXPlugin = Pluggable;
```

Comment:
this type was not correct as it was pointing to list of plugin instead of plugin itself